src/attribute-editor/__tests__/attribute-editor.test.tsx

```diff
@@ -208,6 +215,18 @@ describe('Attribute Editor', () => {
       const wrapper = renderAttributeEditor({ additionalInfo: 'test' });
       expect(wrapper.findAdditionalInfo()!.getElement()).toHaveTextContent('test');
     });
+
+    test('is connected to add button with aria-describedby', () => {
+      const wrapper = renderAttributeEditor({ ...defaultProps, additionalInfo: 'Test Info' });
+      const buttonElement = wrapper.findAddButton().getElement();
+      const info = wrapper.findAdditionalInfo()?.getElement();
+      expect(buttonElement).toHaveAttribute('aria-describedby', info?.id);
+    });
+
+    test('has an ARIA live region', () => {
+      const wrapper = renderAttributeEditor({ ...defaultProps, additionalInfo: 'Test Info' });
+      expect(wrapper.find(`.${liveRegionStyles.root}`)?.getElement()).toBeInTheDocument();
```

Comment:
I would check the content or, if not possible, give some testid to the live region. The component or its subcomponents can have more than one live region.

Reply:
Unfortunately I cannot check the content because of `visible={true}` but I can make the existence check stronger.